antiope-shell/README.md

```diff
@@ -0,0 +1 @@
+# antiope-s3
```

Comment:
antiope-shell?

Reply:
Yeah, for shelling out to the command line.

Reply:
It says `antiope-s3`